### ISSUE TITLE
Endless booting

### ISSUE THREAD

**Issue by MarksenDE** [Nwqda/Sweet-Pineapple-Builder#3]
Hi, this project is really great & awesome! Today i started with an old TP-Link C7 V2 and everything seems fine, but after first boot i saw Welcome/Get started -> Chnges -> The Wifi Pineapple is still booting and it's endless. I've tried to use the recovery utility from TP-Link but with no success. On SSH with root i get no access with password: empty / change_on_install / hak5pineapple and several other. Do you have an idea what's the current password during 1st configuration? Or different user?

Thanks in advance

Mark

**Comment by Nwqda** [comment]
Hello Mark,

Alright, you are almost done.

Can you try to SSH the device (the default password is `root`)

```sh
ssh root@172.16.42.1
root@172.16.42.1's password: root
```

Then you need to hard reset the router to completely remove the previous OpenWrt configuration.
So from the console use this command to erase and reformat the whole JFFS2 partition.

```sh
jffs2reset -y && reboot now
```

After that your device should work correctly.

**Comment by aegisec** [comment]
Hey @Nwqda, I'm getting an endless boot on Buffalo WZR-HP-AG300H as well. SSH is working, but password `root` isn't working. Are there any other passwords that it could default to? 

![endless_boot](https://user-images.githubusercontent.com/93288117/187100972-9767ab9c-4d0e-4b12-bb26-6f8fab8e91e0.png)

![ssh_password_fail](https://user-images.githubusercontent.com/93288117/187101051-a90370fa-b50f-4bf6-b43e-ee2d239c90ec.png)

**Comment by Nwqda** [comment]
Nop, the root account is the only account. Hmm, that's weird, the default password should be root...
Well in that case you have to connect a serial cable to the UART port to manually change the root password.

![WZR-HP-AG300H wifi pineapple](https://user-images.githubusercontent.com/89986678/187103742-cc40e301-3638-4a1f-95b8-e8306a491821.jpg)

**Comment by Nwqda** [comment]
I also invite you to watch this YouTube video which explains all the steps

[![Video PoC WiFI Pineapple Tetra DIY](https://i.ibb.co/7gXHL9q/500px-youtube-social-play.png)](https://www.youtube.com/watch?v=RrzqmJnzbA4)

**Comment by MarksenDE** [comment]
> Hey @Nwqda, I'm getting an endless boot on Buffalo WZR-HP-AG300H as well. SSH is working, but password `root` isn't working. Are there any other passwords that it could default to?
> 
> ![endless_boot](https://user-images.githubusercontent.com/93288117/187100972-9767ab9c-4d0e-4b12-bb26-6f8fab8e91e0.png)
> 
> ![ssh_password_fail](https://user-images.githubusercontent.com/93288117/187101051-a90370fa-b50f-4bf6-b43e-ee2d239c90ec.png)

Yeah that's weird. I've binwalked the firmware and checked the shadow file with hash and the password is correct as 'root'.  
I've tried also to install the original TP-Link fw with tftp but also with no success.

Would be great to hear if someone was successful with a TP-Link router.

Mark
